File: Service_eHospital_DongNai_A/Implement/zzz_Cust_ThamSoService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class zzz_Cust_ThamSoService : BaseService<zzz_Cust_ThamSo, Izzz_Cust_ThamSoRepository>
    , Izzz_Cust_ThamSoService
    {
    private readonly Izzz_Cust_ThamSoRepository _zzz_Cust_ThamSoRepository;
    public zzz_Cust_ThamSoService(Izzz_Cust_ThamSoRepository zzz_Cust_ThamSoRepository) : base(zzz_Cust_ThamSoRepository)
    {
    _zzz_Cust_ThamSoRepository = zzz_Cust_ThamSoRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/zzz_Cust_ThamSo.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { zzz_Cust_ThamSo } from './zzz_Cust_ThamSo.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class zzz_Cust_ThamSoService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'TenThamSo', 'TamNgung'];

    List: zzz_Cust_ThamSo[] | undefined;
    ListFilter: zzz_Cust_ThamSo[] | undefined;
    FormData!: zzz_Cust_ThamSo;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "zzz_Cust_ThamSo";
    }
}

